### PR TITLE
Add Pergamino challenge

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#14.
**Note**: #131 should be merged first than this PR.

### About the challenge:
* This challenge is from the original Brazilian course.
* I made it a progressive challenge, just as #131 
* There is still a final question to be implemented. As soon as I understand it, I'll implement it to the challenge. A new issue must be created for this.

### About `Playwright` test
* The web test is similar to the one in #131, however, this one is easier to follow, as it doesn't have to deal with dialog prompts.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/80920858/126046612-eaa2fb9e-c57c-4c58-8c1d-c6e93cbfd363.mp4

### CODE REVIEW

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
Remember what I mentioned in #131 about the `retos` folder if you'd like to implement it you'd had to change it here as well

File: _sources/lectures/Reto02.rst [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        Primero, las letras de Googlon se clasifican en dos grupos: las letras **z, m, b** se denominan 
+        "letras de tipo zombi", mientras que las demás se conocen como "letras de otro tipo".
+        Los lingüistas han descubierto que las preposiciones en Googlon son palabras que comienzan con una 
+        letra de tipo zombi y terminan con otra letra, es fácil ver que hay 71 preposiciones en el Texto A.
+
+        .. activecode:: ac_r02_2
+            :nocodelens:
+            :include: ac_r02_1
+
+            **Calcule cuántas preposiciones existen en el texto B.** El resultado asígnelo a la varible
```
Comment (LeoCumpli21) [inline _sources/lectures/Reto02.rst:129]:
```suggestion
            **Calcule cuántas preposiciones existen en el texto B.** El resultado asígnelo a la variable 
```

File: _sources/lectures/Reto02.rst [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            kckksgp nzn tpqnm znzppsg tvcgnrb zgdsp tqlqrf vjqqxsp pwj pgft cvl cvr cnhgxsd lkd qlw 
+            vwtbh mfxs gbgw'''.split()
+
+    
+    .. tab:: ac_2
+
+        Estos pergaminos están en el antiguo y misterioso idioma Googlon. Después de muchos años de estudio, 
+        los lingüistas ya conocen algunas características de este idioma.
+        Primero, las letras de Googlon se clasifican en dos grupos: las letras **z, m, b** se denominan 
+        "letras de tipo zombi", mientras que las demás se conocen como "letras de otro tipo".
```
Comment (LeoCumpli21) [inline _sources/lectures/Reto02.rst:121]:
```suggestion
        "letras de tipo zombi", mientras que las demás se conocen como "letras de tipo sobrevivientes".
```

File: _sources/lectures/Reto02.rst [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+
+            myTests().main()
+
+
+    .. tab:: ac_3
+
+        Otro hecho interesante descubierto por los lingüistas es que, en Googlon, los verbos son siempre 
+        palabras de 7 letras que terminan en una letra de tipo otra. Además, si un verbo comienza con
+        una letra de tipo otra, el verbo está en primera persona. Así, leyendo el Texto A, es posible
```
Comment (LeoCumpli21) [inline _sources/lectures/Reto02.rst:155]:
```suggestion
        palabras de 7 letras que terminan en una letra de tipo sobreviviente. Además, si un verbo comienza con
        una letra de tipo sobreviviente, el verbo está en primera persona. Así, leyendo el Texto A, es posible 
```

**Review by NicolasSandoval — APPROVED** [review]
Great job! Thanks.

**Pull request merged**